[31m

processing file: repro.qmd
[39m1/41                   
2/41 [rwf]             
3/41                   
4/41 [init_git4]       
5/41                   
6/41 [here_01]         
7/41                   
8/41 [here_02]         
9/41                   
10/41 [random 1]        
11/41                   
12/41 [random 2]        
13/41                   
14/41 [random 3]        
15/41                   
16/41 [random 4]        
17/41                   
18/41 [introcmp01]      
19/41                   
20/41 [introcmp02]      
21/41                   
22/41 [introcmp03]      
23/41                   
24/41 [tb1]             
25/41                   
26/41 [introcmp04]      
27/41                   
28/41 [tb2]             
29/41                   
30/41 [introcmp05]      
31/41                   
32/41 [tb3]             
33/41                   
34/41 [dt_01]           
[31mError in `file()`:
! cannot open the connection
Backtrace:
  1. global .main()
  2. execute(...)
  3. rmarkdown::render(...)
  4. knitr::knit(knit_input, knit_output, envir = envir, quiet = quiet)
  5. knitr:::process_file(text, output)
     ...
 16. base::withRestarts(...)
 17. base (local) withRestartList(expr, restarts)
 18. base (local) withOneRestart(withRestartList(expr, restarts[-nr]), restarts[[nr]])
 19. base (local) docall(restart$handler, restartArgs)
 21. evaluate (local) fun(base::quote(`<smplErrr>`))
[39m[31m


Quitting from lines 827-844 [dt_01] (repro.qmd)
[39m[31mExecution halted
[39m